### ISSUE TITLE
Update weights and biases

### ISSUE THREAD

**Issue by kevinsuedmersen** [kevinsuedmersen/library-neuralnetworksfromscratch#3]
- [x] Use the gradients calculated during backpropagation to update the weights and biases using Stochastic Gradient Descent
- Test that the weights and biases are updated into the right direction
    - [x] In `test_model.py`, test that the cost decreases after each train step with the same input data
